include/IrrCompileConfig.h

```diff
@@ -89,12 +89,6 @@
 #endif
 
 
-//! Maximum number of textures and input images we can feed to a shader
-/** These limits will most likely be below your GPU hardware limits
-**/
-#define _IRR_MATERIAL_MAX_TEXTURES_ 8
-#define _IRR_MATERIAL_MAX_IMAGES_ 8
-
 //! Maximum of other shader input (output) slots
```

Comment:
I'm not sure that we need the other _MAX_ down here on the next 10 lines

Reply:
me too thats why i leaved them :P

Reply:
resolved